src/main/java/io/snowdrop/github/reporting/model/IssueDTO.java

```diff
@@ -2,10 +2,16 @@
 
 import java.util.Date;
 
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
 import org.eclipse.egit.github.core.Issue;
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
 
-public class IssueDTO {
+@Entity
+public class IssueDTO extends PanacheEntityBase {
```

Comment:
It's kind of weird to name an entity `DTO`, isn't?

Reply:
Well, it was a DTO before becoming an entity ;-)

Let me see, if I can waste the suffix without clashing with egit types.

Reply:
@geoand renamed!